### DISCUSSION TITLE
"Publish to Marketplace" checkbox no longer appears - can't update Action with new release

### DISCUSSION THREAD

**Discussion by alstr** [community/community#26787] [Actions]
<p>I released an Action on the Marketplace a while ago and updated it a couple of times:</p>
<p><a href="https://github.com/marketplace/actions/todo-to-issue" rel="nofollow noopener">https://github.com/marketplace/actions/todo-to-issue</a></p>
<p><a href="https://github.com/alstr/todo-to-issue-action" rel="nofollow noopener">https://github.com/alstr/todo-to-issue-action</a></p>
<p>I’ve just drafted a new release but the “Publish to Marketplace” checkbox is no longer present so I can’t update it.</p>
<p>I have no idea why.</p>

**Comment by Yanjingzhu** [comment]
<p>Normally, when there is a new release version in the Action repo, the Action in marketplace will update to the latest version automatically.</p>
<p>For your issue, could you please try to click on the <strong>Delist</strong> button in the marketplace page? </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/2cdd5eefea6b9bcfe32632c4b897aec572ac56db.png" data-download-href="/uploads/short-url/6oTicqm1cfB3d4dKOXEnQTJ3Nbl.png?dl=1" title="Delist.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181099907-bd51b522-0ae1-4b25-9a0d-c020a9e8a26d.png" alt="Delist.png" data-base62-sha1="6oTicqm1cfB3d4dKOXEnQTJ3Nbl" width="690" height="218" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/2cdd5eefea6b9bcfe32632c4b897aec572ac56db_2_690x218.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/2cdd5eefea6b9bcfe32632c4b897aec572ac56db_2_1035x327.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/2cdd5eefea6b9bcfe32632c4b897aec572ac56db.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/2cdd5eefea6b9bcfe32632c4b897aec572ac56db_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Delist.png</span><span class="informations">1176×372 24.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Then you can go to your action repo , there will be “Publish this Action to Marketplace” message . Please draft a release and publish it to marketplace again, can you see the new release in marketplace then?    </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/f017fcbb75ad773b85cf56de3c504ca2fb2a0589.png" data-download-href="/uploads/short-url/yfXZ5zY193TMvlBTYdsr5HszYaB.png?dl=1" title="publish Action to marketplace.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181099908-9b83c195-7156-414a-ba33-b433fb377ed6.png" alt="publish Action to marketplace.png" data-base62-sha1="yfXZ5zY193TMvlBTYdsr5HszYaB" width="690" height="355" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/f017fcbb75ad773b85cf56de3c504ca2fb2a0589_2_690x355.png, https://aws1.discourse-cdn.com/github/optimized/2X/f/f017fcbb75ad773b85cf56de3c504ca2fb2a0589_2_1035x532.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/f/f017fcbb75ad773b85cf56de3c504ca2fb2a0589.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/f017fcbb75ad773b85cf56de3c504ca2fb2a0589_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">publish Action to marketplace.png</span><span class="informations">1035×533 37.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by alstr** [comment]
<p>Hello <a class="mention" href="/yanjingzhu">@yanjingzhu</a>,</p>
<p>I don’t think I should have to do this, but I gave it a try anyway.</p>
<p>I delisted the Action from the Marketplace. The message appears on the repo homepage telling me to draft a release:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d91c6e446ba6dd4e6202d85b9199901fb6d15ceb.png" data-download-href="/uploads/short-url/uYEwsUar40JdIpmmi8Xo9JvB8mf.png?dl=1" title="Screenshot 2020-03-23 at 10.08.41.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/11612438/181099909-51c6fdf6-215c-4473-93e6-9a1295fe8910.png" alt="Screenshot 2020-03-23 at 10.08.41.png" data-base62-sha1="uYEwsUar40JdIpmmi8Xo9JvB8mf" width="690" height="232" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d91c6e446ba6dd4e6202d85b9199901fb6d15ceb_2_690x232.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/d91c6e446ba6dd4e6202d85b9199901fb6d15ceb_2_1035x348.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/d/d91c6e446ba6dd4e6202d85b9199901fb6d15ceb_2_1380x464.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d91c6e446ba6dd4e6202d85b9199901fb6d15ceb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-03-23 at 10.08.41.png</span><span class="informations">2020×680 74.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Upon clicking ‘Draft a release’, there is still no option to publish it to Marketplace:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/fb30b3f3d76bd33cb91692efa677b0ea471a7fc2.png" data-download-href="/uploads/short-url/zQ8bOR49A5w9Z43c0vsreJpBwmC.png?dl=1" title="Screenshot 2020-03-23 at 10.08.55.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/11612438/181099910-3f20c000-3bc7-4a73-a4d6-a2568e84f7eb.png" alt="Screenshot 2020-03-23 at 10.08.55.png" data-base62-sha1="zQ8bOR49A5w9Z43c0vsreJpBwmC" width="541" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/fb30b3f3d76bd33cb91692efa677b0ea471a7fc2_2_541x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/f/fb30b3f3d76bd33cb91692efa677b0ea471a7fc2_2_811x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/f/fb30b3f3d76bd33cb91692efa677b0ea471a7fc2_2_1082x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/fb30b3f3d76bd33cb91692efa677b0ea471a7fc2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-03-23 at 10.08.55.png</span><span class="informations">1540×1422 154 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I even drafted the release to see if it did anything, but it didn’t.</p>
<p>So I still can’t publish the updated Action - and now my Action has disappeared from Marketplace completely. That is what I expected to happen when you suggested doing this.</p>
<p>Please could someone look into this more closely as clearly something is wrong on your end? I published my Action no problem a few weeks ago and there is no obvious reason I can see why I shouldn’t be able to update it (or now publish it afresh)…</p>

**Comment by Yanjingzhu** [comment]
<p><a class="mention" href="/alstr">@alstr</a> Could you please check whether you could see the publish to marketplace checkbox in your other Action repos? Is this issue just happened in “todo-to-issue-action” ? </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/0eb3213bd9a65deb8860b44aca715a903caff8af.png" data-download-href="/uploads/short-url/262t2JPUcjIUNPPrOxF1RiYVFHF.png?dl=1" title="publish to marketplace.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181099911-b13b20c5-dc38-450a-bb47-53e921189565.png" alt="publish to marketplace.png" data-base62-sha1="262t2JPUcjIUNPPrOxF1RiYVFHF" width="489" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/0eb3213bd9a65deb8860b44aca715a903caff8af_2_489x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/0eb3213bd9a65deb8860b44aca715a903caff8af_2_733x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/0eb3213bd9a65deb8860b44aca715a903caff8af.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/0eb3213bd9a65deb8860b44aca715a903caff8af_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">publish to marketplace.png</span><span class="informations">781×797 25.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>For you getting support from engineering team directly, you could contact github support. <a href="https://support.github.com/contact" rel="nofollow noopener">https://support.github.com/contact </a></p>

**Comment by alstr** [comment]
<p><span class="mention">@yanjingzhuIf</span> I create a new repo it will let me draft a release and publish to Marketplace.</p>
<p>I don’t want to do that though. I want to retain the existing repo.</p>
<p>I have contacted GitHub support but no reply as yet.</p>

**Comment by pqt** [comment]
<p>I’ve got a similar issue, except I don’t see the checkbox at all. Even on new repos. Hopefully this is a temporary thing, I’ve also reached out to GitHub support but it’s night time at HQ so I would imagine an email response would show up sometime Thursday/Friday.</p>
<p>Looking forward to seeing answers on this.</p>

**Comment by Yanjingzhu** [comment]
<p><a class="mention" href="/alstr">@alstr</a> <a class="mention" href="/pqt">@pqt</a>  Sorry for the unhappy experience you’ve met. I found an internal ticket which reported this “unable to publish to marketplace” issue. And engineering team is working on it now. When there’s any good news, I will update here at the first time. </p>

**Comment by pqt** [comment]
<p>Good to know. It would be nice if there were some more indicators of successful recognition of the repo being a correct sequence for being an action in the future (just feedback that can maybe be passed along)</p>
<p>I checked, rechecked, re-rechecked and made a couple whole new repos with formats just to try and get an option to show up.</p>
<p>I’d love something like pages where you have a settings section where you can conveniently apply the modifier that this repo is slated for the actions marketplace or something.</p>
<p>If there’s a better feedback channel let me know and I’ll send it there instead.</p>
<p>Hopefully this isn’t too delayed, currently a big need at work!</p>

**Comment by Yanjingzhu** [comment]
<p><a class="mention" href="/pqt">@pqt</a> I would recommend you to share your ideas in the <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="nofollow noopener">Feedback form for GitHub Actions</a>. </p>

**Comment by robertdebock** [comment]
<p>I had the same issue; turned out I had a YAML error. Pasted my action.yml in <a href="http://www.yamllint.com/" rel="nofollow noopener">http://www.yamllint.com/</a> , fixed the error and the marketplace release buttons start appearing again.</p>
<p>Good luck!</p>

**Reply by amitsadafule** [reply]
thanks @robertdebock . It helped me. There was an invalid character in my yml file.

**Comment by alstr** [comment, marked as answer]
<p>In the end the GitHub team had to perform a manual action to make the buttons reappear. Last I heard they were still investigating the root cause. :man_shrugging:</p>

**Comment by Anthonykung** [comment]
<p>I have the same issue too and it was because I added two extra indents, it was so hard to see…</p>

**Reply by gilzonme** [reply]
Yeah, it happened for me today!
